test_utils.py

```diff
@@ -36,14 +33,6 @@ def expected_len(region_name):
     return region_info.end - region_info.start + 1
 
 
-def test_region_dict():
```

Comment:
If we need to test the data we using to test our code there is no point in using the data in the first place. If the data is wrong the tests will fail how hard is that?